config/webpack.config.prod.js

```diff
@@ -230,7 +230,7 @@ module.exports = [{
     new RollbarSourceMapPlugin({
       accessToken: process.env.ROLLBAR_TOKEN || 'notoken',
       version: process.env.CIRCLE_SHA1 || 'noversion',
-      publicPath: publicPath
+      publicPath: 'https://dm-frontend.apps.platform.digital.gov.au/bundle/'
```

Comment:
these don't look right

Reply:
turns out rollbar wants paths that start with http for a variable called "minified_url" https://rollbar.com/docs/api/sourcemaps/ but the plugin doesn't mention that https://github.com/thredup/rollbar-sourcemap-webpack-plugin/blob/master/src/RollbarSourceMapPlugin.js#L99

Reply:
well at v least make it a var rather copy/pasting so many times